### ISSUE TITLE
Error: No package metadata was found

### ISSUE THREAD

**Issue by igor-kurenkov** [danilodelucio/NukeDiffusion#5]
Successfully installed everything (i guess). Run Nuke, create NukeDiffusion node, pick SD checkpoind, add basic prompt, hit Generate Image button - i  get this output in console:
```
----------------------------------------------------------------------------------

                              NukeDiffusion Terminal

----------------------------------------------------------------------------------
┌──────────────────────────┬─────────────────────────────────────────────────────┐
│ - Python Version:        │ 3.11.6                                              │
├──────────────────────────┼─────────────────────────────────────────────────────┤
│ - Torch version:         │ 2.2.2+cpu                                           │
├──────────────────────────┼─────────────────────────────────────────────────────┤
│ - Workflow:              │ txt2img                                             │
├──────────────────────────┼─────────────────────────────────────────────────────┤
│ - Checkpoint:            │ epicrealism_naturalSinRC1VAE.safetensors            │
├──────────────────────────┼─────────────────────────────────────────────────────┤
│ - SD Model:              │ SD                                                  │
├──────────────────────────┼─────────────────────────────────────────────────────┤
│ - Positive Prompt:       │ cyberpunk car                                       │
├──────────────────────────┼─────────────────────────────────────────────────────┤
│ - Negative Prompt:       │                                                     │
├──────────────────────────┼─────────────────────────────────────────────────────┤
│ - Resolution:            │ 512x512                                             │
├──────────────────────────┼─────────────────────────────────────────────────────┤
│ - Seed:                  │ -1                                                  │
├──────────────────────────┼─────────────────────────────────────────────────────┤
│ - CFG:                   │ 7.0                                                 │
├──────────────────────────┼─────────────────────────────────────────────────────┤
│ - Steps:                 │ 20                                                  │
└──────────────────────────┴─────────────────────────────────────────────────────┘

⚠  Using a local checkpoint: epicrealism_naturalSinRC1VAE.safetensors.


- It was not possible to generate your image!

- Error: No package metadata was found for The 'requests' distribution was not found and is required by this
application.
Try: pip install transformers -U or pip install -e '.' if you're working with git main.
```

Tried different checkpoints, doesn't help.
Tried both pip install transformers -U and pip install -e '.'. Didn't help either

**Comment by danilodelucio** [comment]
Hi Igor!

Did you download NukeDiffusion from the **main** branch? I see you are using Torch for CPU, which is not supported for this NukeDiffusion version.

If possible, try to download NukeDiffusion from the **linux** branch please:
https://github.com/danilodelucio/NukeDiffusion/tree/linux

I will merge to the **main** soon, is because I'm still working on Linux/Mac support, but for the CPU support should work for you.

Feel free to reach out, thanks!

**Comment by igor-kurenkov** [comment]
> Hi Igor!
> 
> Did you download NukeDiffusion from the **main** branch? I see you are using Torch for CPU, which is not supported for this NukeDiffusion version.
> 
> If possible, try to download NukeDiffusion from the **linux** branch please: https://github.com/danilodelucio/NukeDiffusion/tree/linux
> 
> I will merge to the **main** soon, is because I'm still working on Linux/Mac support, but for the CPU support should work for you.
> 
> Feel free to reach out, thanks!

I'm on Windows so yes, i downloaded the main branch. are you sure I need the linux version??
While installing i had warning about the torch version. I had already had 2.2.2 installed and the warning said i need 2.2.0, so i probably found the wrong 2.2.0 to install

**Comment by danilodelucio** [comment]
Yes, even the branch is called **linux**, it's where I am improving the tool and there is a new structure folder for Windows and Linux/Mac. Also, I've improved the installation setup, so you just need to run **_NUKEDIFFUSION_SETUP.bat** file only.

Regarding the Torch issue, it's tricky to say. Some libraries from Stable Diffusion try to install a Torch version which is not compatible with some requirements, so please ignore it (I tried my best to handle with it).

In summary, try to install again using the **linux** branch, and let me know if this works. Also feel free to send some screenshots if needed.

**Comment by igor-kurenkov** [comment]
Thanks a lot, I got it working with the linux branch!
